code/modules/clothing/factions/dominia.dm

```diff
@@ -261,3 +261,85 @@
 	item_state = "dom_dress"
 	contained_sprite = TRUE
 
+/obj/item/clothing/accessory/poncho/dominia/surcoat
+	name = "tribunalist surcoat"
+	desc = "A simple red surcoat commonly worn by Dominian clergy members."
+	desc_fluff = "Spun with rough but hardy fabrics from the Dominian frontier, this surcoat is commonly worn by poorer Tribunal clergy as well as missionaries\
+	seeking protection from the elements. This garment was popularized by the Kael'kah sect and remains respected as a symbol of humility and poverty amongst priests."
+	icon = 'icons/clothing/suits/capes/dominia_surcoat.dmi'
+	icon_state = "dominian_surcoat"
+	item_state = "dominian_surcoat"
+	overlay_state = "dominian_surcoat"
+	icon_override = null
+	contained_sprite = TRUE
```

Comment:
If the dominian parent accessory already has contained sprite you dont need to copypaste here

Reply:
There is no parent

Reply:
There is now.